third_party/terraform/resources/resource_compute_instance_template.go

```diff
@@ -888,3 +865,28 @@ func resourceComputeInstanceTemplateDelete(d *schema.ResourceData, meta interfac
 	d.SetId("")
 	return nil
 }
+
+func expandResourceComputeInstanceTemplateScheduling(d *schema.ResourceData, meta interface{}) (*computeBeta.Scheduling, error) {
+	v, ok := d.GetOk("scheduling")
+	if !ok || v == nil {
+		return &computeBeta.Scheduling{
+			OnHostMaintenance: "MIGRATE",
+		}, nil
+	}
+
+	schedulings := v.([]interface{})
+	if len(schedulings) > 1 {
+		return nil, fmt.Errorf("Unable to expand scheduling, more than one `scheduling` block is defined")
```

Comment:
It looks like the instance template schema doesn't put MaxItems: 1 for scheduling, but instance does. Should the schema be updated to match?

Reply:
yes!